.github/workflows/ios-unit-tests.yml

```diff
@@ -1,15 +1,16 @@
-name:                         E2E Test
+name:                         iOS Unit Tests
 
 on:
   pull_request:
   push:
     branches:
       - master
       - stable
+
 jobs:
-  check:
-    name:                     Check on MacOS
```

Comment:
`Check on MacOS` name is marked as a required action for this repo. Without it, it's not possible to merge any PRs. The name can be changed in the settings, but I might prefer the old name. I think`Run iOS Unit Tests` doesn't hint that linter checks also run there.